### ISSUE TITLE
Comment on changes for public body entries

### ISSUE THREAD

**Issue by codedust** [okfde/froide#441]
When requesting changes for public body entries, it might be useful to add a field for notes/references to explain why the change was made and/or provide information about the source of the updated information.

https://github.com/okfde/froide/blob/b01102b1434e3e933f1d02426aa2771cb831a3f8/froide/publicbody/forms.py#L68

**Comment by rugk** [comment]
Indeed, that’s also useful e.g. if you want to argue that an agency should be removed or so.
